yarn.lock

```diff
@@ -401,7 +401,7 @@
     debug "^4.1.1"
     semver "^7.3.2"
 
-"@oclif/config@^1", "@oclif/config@^1.12.12", "@oclif/config@^1.15.1":
+"@oclif/config@^1.12.12", "@oclif/config@^1.15.1", "@oclif/config@^1.17.0":
```

Comment:
I have just a silly question here, are we pushing yarn.lock changes? What i mean, we're using this file for the CircleCi tests?

Reply:
@rorolopetegui We include the yarn.lock to ensure dependencies are consistent across machines: https://classic.yarnpkg.com/en/docs/yarn-lock/#toc-check-into-source-control

So to answer, your question: yes, the yarn.lock is respected in the CircleCI tests